### PR TITLE
MAT 5 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/metropolis_reformer.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Name:Metropolis Reformer
+ManaCost:2 W
+Types:Creature Angel Cleric
+PT:2/3
+K:Flying
+K:Vigilance
+S:Mode$ Continuous | Affected$ You | AddKeyword$ Hexproof | Description$ You have hexproof.
+T:Mode$ DamageDoneOnce | ValidTarget$ Card.Self | Execute$ TrigGain | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME is dealt damage, you gain that much life
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/metropolis_reformer.txt:8]: . missing

File: forge-gui/res/cardsfolder/upcoming/nashi_moons_legacy.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Name:Nashi, Moon's Legacy
+ManaCost:B G U
+Types:Legendary Creature Rat Shaman
+PT:3/4
+K:Menace
+K:Ward:1
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ DBExile | TriggerDescription$ Whenever CARDNAME attacks, exile up to one target legendary or Rat card from your graveyard and copy it. You may cast the copy. (You still pay its costs. A copy of a permanent spell becomes a token.)
+SVar:DBExile:DB$ ChangeZone | DefinedPlayer$ You | TargetMin$ 0 | TargetMax$ 1 | Destination$ Exile | ChangeNum$ 1 | ValidTgts$ Rat.YouCtrl,Card.Legendary+YouCtrl | TgtPrompt$ Select target legendary or Rat card from your graveyard | Origin$ Graveyard | Hidden$ True | Mandatory$ True | RememberChanged$ True | SubAbility$ DBPlay
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/nashi_moons_legacy.txt:8]: Why do it as Hidden?!

File: forge-gui/res/cardsfolder/upcoming/nashi_moons_legacy.txt [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+SVar:DBPlay:DB$ Play | Valid$ Card.IsRemembered | ValidZone$ Exile | Controller$ You | CopyCard$ True | ValidSA$ Spell | ValidSA$ Spell | Optional$ True | CopyCard$ True | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/nashi_moons_legacy.txt:9]: Defined is easier than Valid

File: forge-gui/res/cardsfolder/upcoming/nissa_resurgent_animist.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Name:Nissa, Resurgent Animist
+ManaCost:2 G
+Types:Legendary Creature Elf Scout
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Land.YouCtrl | TriggerZones$ Battlefield | Execute$ TrigMana | TriggerDescription$ Landfall — Whenever a land enters the battlefield under your control, add one mana of any color. Then if this is the second time this ability has resolved this turn, reveal cards from the top of your library until you reveal an Elf or Elemental card. Put that card into your hand and the rest on the bottom of your library in a random order.
+SVar:TrigMana:DB$ Mana | Produced$ Any | AILogic$ MostProminentInComputerHand | SubAbility$ RevealUntil
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/nissa_resurgent_animist.txt:6]: This AILogic makes no sense at all

File: forge-gui/res/cardsfolder/upcoming/reckless_handling.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Name:Reckless Handling
+ManaCost:1 R
+Types:Sorcery
+A:SP$ ChangeZone | Cost$ R | Origin$ Library | Destination$ Hand | ChangeType$ Card | ChangeNum$ 1 | SubAbility$ DBDiscard | Mandatory$ True | SpellDescription$ Search your library for an artifact card, reveal it, put it into your hand, shuffle, then discard a card at random. If an artifact card was discarded this way, Reckless Handling deals 2 damage to each opponent.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/reckless_handling.txt:4]: Why Mandatory?

Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/reckless_handling.txt:4]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/nissa_resurgent_animist.txt (hunk 4 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/nissa_resurgent_animist.txt:6]: I copied it from Lotus Cobra, you want it removed as well ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/nissa_resurgent_animist.txt:6]: yes thanks!

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/nissa_resurgent_animist.txt:6]: Actually wait, it does make sense
It just seems to be missing the implementation for it

File: forge-gui/res/cardsfolder/upcoming/nashi_moons_legacy.txt [hunk 6]
```diff
@@ -0,0 +1,15 @@
+Name:Nashi, Moon's Legacy
+ManaCost:B G U
+Types:Legendary Creature Rat Shaman
+PT:3/4
+K:Menace
+K:Ward:1
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ DBExile | TriggerDescription$ Whenever CARDNAME attacks, exile up to one target legendary or Rat card from your graveyard and copy it. You may cast the copy. (You still pay its costs. A copy of a permanent spell becomes a token.)
+SVar:DBExile:DB$ ChangeZone | DefinedPlayer$ You | TargetMin$ 0 | TargetMax$ 1 | Destination$ Exile | ChangeNum$ 1 | ValidTgts$ Rat.YouCtrl,Card.Legendary+YouCtrl | TgtPrompt$ Select target legendary or Rat card from your graveyard | Origin$ Graveyard | RememberChanged$ True | SubAbility$ DBPlay
+SVar:DBPlay:DB$ Play | Defined$ Remembered | ValidZone$ Exile | Controller$ You | CopyCard$ True | ValidSA$ Spell | Optional$ True | CopyCard$ True | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/nashi_moons_legacy.txt:9]: ValidZone$ also obsolete

File: forge-gui/res/cardsfolder/upcoming/nashi_moons_legacy.txt (hunk 2 above)
Comment (squee1968) [reply forge-gui/res/cardsfolder/upcoming/nashi_moons_legacy.txt]: I think TargetMin$ should be 1

File: forge-gui/res/cardsfolder/upcoming/nashi_moons_legacy.txt (hunk 3 above)
Comment (squee1968) [reply forge-gui/res/cardsfolder/upcoming/nashi_moons_legacy.txt]: You have ValidSA$ Spell twice

File: forge-gui/res/cardsfolder/upcoming/reckless_handling.txt [hunk 7]
```diff
@@ -0,0 +1,10 @@
+Name:Reckless Handling
+ManaCost:1 R
+Types:Sorcery
+A:SP$ ChangeZone | Cost$ R | Origin$ Library | Destination$ Hand | ChangeType$ Card | ChangeNum$ 1 | SubAbility$ DBDiscard | SpellDescription$ Search your library for an artifact card, reveal it, put it into your hand, shuffle, then discard a card at random. If an artifact card was discarded this way, CARDNAME deals 2 damage to each opponent.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/reckless_handling.txt:4]: Wrong Type

**Review by tool4ever — APPROVED** [review]

**Pull request merged**